### PR TITLE
Occupant updates

### PR DESCRIPTION
**Pull request by ahmedalmoraly**

### CODE REVIEW

File: vision/shared/PlottingBodyListener.h [hunk 1]
```diff
@@ -132,17 +132,6 @@ class PlottingBodyListener : public vision::BodyListener, public PlottingListene
         image_data_ = viz_.getImageData();
     }
 
-    using PlottingListener::processResults; // make the overload taking a Frame arg visible
```
Comment (wiperz1789) [inline vision/shared/PlottingBodyListener.h:135]: @ahmedalmoraly I remember  we did this to mitigate webcam issue where swapping was happening ?

File: vision/shared/PlottingBodyListener.h [hunk 2]
```diff
@@ -36,11 +32,7 @@ static const std::vector<BodyPoint> BODY_POINTS = {
     BodyPoint::LEFT_ELBOW,
     BodyPoint::LEFT_WRIST,
     BodyPoint::RIGHT_HIP,
-    BodyPoint::RIGHT_KNEE,
```
Comment (adennie) [inline vision/shared/PlottingBodyListener.h:39]: note that NECK moved to the end, so be careful when you merge.  That was fixed in Umang's branch that I just merged.

File: vision/shared/PlottingBodyListener.h (hunk 1 above)
Comment (ahmedalmoraly) [reply vision/shared/PlottingBodyListener.h:135]: yes, we removed that function and now using the `processResults(frame)` overload

File: vision/shared/PlottingBodyListener.h (hunk 2 above)
Comment (ahmedalmoraly) [reply vision/shared/PlottingBodyListener.h:39]: I don't think the order matters in here. but I'll update it

**Pull request merged**